Tests/ApolloCodegenTests/CodeGeneration/Templates/SelectionSet/SelectionSetTemplateTests.swift

```diff
@@ -7407,7 +7407,7 @@ class SelectionSetTemplateTests: XCTestCase {
     expect(rendered_allAnimals_deferredAsRoot).to(equalLineByLine(
       """
       /// AllAnimal.Root
-      public struct Root: TestSchema.InlineFragment, ApolloAPI.Deferrable {
+      public struct Root: TestSchema.InlineFragment {
```

Comment:
Do these tests even serve a purpose anymore? If we aren't adding the `Deferrable` protocol conformance, then what behavior are we testing?

Reply:
Yes I see your point; I'll handle this in the next PR (https://github.com/apollographql/apollo-ios-dev/pull/219) which deals with codegen.